command/resource/client/helper.go

```diff
@@ -91,3 +106,195 @@ func (t *TValue[T]) Merge(onto *T) error {
 	}
 	return nil
 }
+
+type OuterResource struct {
+	ID         *ID            `json:"id"`
+	Owner      *ID            `json:"owner"`
+	Generation string         `json:"generation"`
+	Version    string         `json:"version"`
+	Metadata   map[string]any `json:"metadata"`
+	Data       map[string]any `json:"data"`
+}
+
+type Tenancy struct {
+	Partition string `json:"partition"`
+	Namespace string `json:"namespace"`
+	PeerName  string `json:"peerName"`
```

Comment:
You'll probably have to remove `PeerName` since it is no longer part of the `pbresource.Tenancy`.

Reply:
Yup, removed all `peer` things in this resource cli